src/Nethermind/Ethereum.Test.Base/AuthorizationListJson.cs

```diff
@@ -10,7 +11,7 @@ public class AuthorizationListJson
     public Address Address { get; set; }
     public ulong Nonce { get; set; }
     public ulong V { get; set; }
```

Comment:
Should it be `byte`?

Reply:
Yes, this actually shouldn't be V but yParity/RecoveryId